extensions/crc/src/crc-cli.ts

```diff
@@ -0,0 +1,169 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+import type { ChildProcess } from 'node:child_process';
+import { spawn } from 'node:child_process';
+import { isMac, isWindows } from './util';
+
+import type { Logger } from '@tmpwip/extension-api';
+
+const macosExtraPath = '/usr/local/bin:/opt/local/bin';
+
+let daemonProcess: ChildProcess;
+
+export function getInstallationPath(): string {
+  const env = process.env;
+  if (isWindows()) {
+    return `c:\\Program Files\\Red Hat OpenShift Local;${env.PATH}`;
```

Comment:
Make this a constant; very likely to change or might be multiple (due to community release)